docs/dev/connectors/streamfile_sink.md

```diff
@@ -765,6 +765,7 @@ and Flink will throw an exception.
 normal job termination (*e.g.* finite input stream) and termination due to failure, upon normal termination of a job, the last 
 in-progress files will not be transitioned to the "finished" state.
 
+TODO I think this is wrong.
```

Comment:
Why this might be wrong ?

Reply:
For example [here](https://github.com/apache/flink/blob/master/flink-filesystems/flink-hadoop-fs/src/main/java/org/apache/flink/runtime/fs/hdfs/HadoopRecoverableFsDataOutputStream.java#L299) we do not seem to fail. Although S3 fails (https://github.com/apache/flink/blob/master/flink-filesystems/flink-s3-fs-base/src/main/java/org/apache/flink/fs/s3/common/writer/S3Committer.java#L100), so the comment is correct.

Reply:
Very thanks for the explanation! From the code it seems if the src does not exists & dest exists, it would skip the file, but if the dest does not exist, s3 would throw exceptions while Hadoop ignore the error. Thus I'm a bit confused here that would we still keep the original notes ? it seems the sink indeed do not throw exception for all the in-progress files not found.

Reply:
Yes, I think we should definitely make the behaviour uniform. This is a bug and we should open a JIRA for that and this is why I put the comment, so that the reviewers can also have a look and verify. But for the docs, I think we should simply have the "least common denominator" which is hard failure in this case. Feel free to open an issue for this and potentially point to the discussion in this PR :)

Reply:
Got it, very thanks for the explanation! I would open the issue~